projects/core/src/lib/types/mask-processors.ts

```diff
@@ -5,7 +5,7 @@ export type MaskPreprocessor = (
         elementState: ElementState;
         data: string;
     },
-    actionType: 'deleteBackward' | 'deleteForward' | 'insert' | 'validation',
+    actionType: MaskPreprocessorAction,
```

Comment:
String union type has equal strength of typing as enum (without increasing library' bundle as enum does).

@RomanBorisov 
Do you propose this change because you want to DX ? 🤔

Reply:
@waterplea @vladimirpotekhin @splincode 

What do you think about it? Could we afford to increase bundle by extra object ? :)

Reply:
If main problem is bundle size, I can rewrite enum defenition like this
`export const enum MaskPreprocessorAction`
When we use const target enum does not appear in prod bundle. Example 
https://www.typescriptlang.org/play?#code/MYewdgzgLgBApmArgWxgZQJbIA4Bs4CiSqA3jAFAxUwCCMAvDAOQCGTANJdQEIPMBGHLlQDCfJsCbkAvuXL5YLAFzoseQsT6Yc+IigB0NIA

Reply:
> I can rewrite enum defenition like this

We can't export `const enum` from lib, unfortunately  https://ncjamieson.com/dont-export-const-enums/

> Could we afford to increase bundle by extra object ? :)

I think this particular object will not noticeably affect the weight, but I would prefer to use [only union type](https://www.bam.tech/article/should-you-use-enums-or-union-types-in-typescript) in the project, so that everything is uniform and not to bother with this in the future

Reply:
@RomanBorisov We discussed this moment with the team.

Let's revert changes with `MaskitoPreprocessorAction` enum and keep union type (as it was).

Other changes is useful, let's keep them:
* projects/core/src/lib/utils/get-not-empty-selection.ts ✅ 
* projects/core/src/lib/utils/test/get-not-empty-selection.spec.ts ✅ 

___

About `KeyboardKeys` enum 🤔 
I think it is better to improve typings in same way as `TypedInputEvent`:
https://github.com/Tinkoff/maskito/blob/ff48b73926205c36c78d88f0c1d1fc320f054533/projects/core/src/lib/types/typed-input-event.ts#L1-L4

We can introduce `TypedKeyboardEvent`.
But let's do it for separate PR.